src/scss/views/widgets/_spinner.scss

```diff
@@ -0,0 +1,79 @@
+.spinner {
+  min-width: 50px;
+  height: 20px;
+  text-align: center;
+  font-size: 10px;
+
+  & > div {
+    background-color: $color-white;
+    height: 100%;
+    width: 3px;
+    margin-right: 1px;
+    display: inline-block;
+
+    -webkit-animation: sk-stretchdelay 1.2s infinite ease-in-out;
```

Comment:
hah! awesome. `sk-stretchdelay`, never seen that animation type before!

edit: oh! you define it below, even cooler.